railway/railway.md

```diff
@@ -0,0 +1,81 @@
+---
+Documentation for Railway Template for Coder
+https://railway.app/template/cUQ8_P?referralCode=tfH8Uw
+---
+
+<div align="center">
+  <a href="https://coder.com#gh-dark-mode-only">
+    <img src="https://raw.githubusercontent.com/coder/coder/main/docs/images/logo-white.png" style="width: 128px">
+  </a>
+  <h3>
+  Self-Hosted Remote Development Environments
+  </h3>
+  <a href="https://coder.com">
+    <img style="width: 650px" src="https://raw.githubusercontent.com/coder/coder/main/docs/images/banner-white.png">
+  </a>
+</div>
+
+[Coder](https://coder.com) enables organizations to set up development environments in the cloud. Environments are defined with Terraform, connected through a secure high-speed Wireguard® tunnel, and are automatically shut down when not in use to save on costs. Coder gives engineering teams the flexibility to use the cloud for workloads that are most beneficial to them.
+
+- Define development environments in Terraform
+  - EC2 VMs, Kubernetes Pods, Docker Containers, etc.
+- Automatically shutdown idle resources to save on costs
+- Onboard developers in seconds instead of days
+
+## Quickstart
+
+### 1. Fill the environment variables
+
+The current version of the template allows you to set an optional environment variable `CODER_WILDCARD_ACCESS_URL` which will allow you to forward ports from your workspace to a custom wildcard subdomain of the URL you provide. This is useful if you want to access a service running in your workspace from a browser. For example, if you set `CODER_WILDCARD_ACCESS_URL` to `*.coder.example.com`, you can access a service running on port 8080 in your workspace by visiting `8080--app-name-workspace-name-.coder.example.com` in your browser.
```

Comment:
Mention that you have to use a custom domain with Railway and cannot use railway's built-in domain for this wildcard.

Reply:
Linking do docs does not help here as on Railway only way to set the flags is through Env variables.